andela/ah-orcas-frontend
# #162888665 fix update/delete article

#### What does this PR do?
Allow user to view and create articles
#### Description of Task to be completed?
``` Add dialog box to delete button```
``` fix update article```
``` fix delete/update button view```

#### How should this be manually tested?
- Clone this repository
`git clone https://github.com/andela/ah-orcas-frontend.git`
- Change the working directory to the project folder
`cd ah-orcas-frontend`
- Checkout to the appropriate branch
`ch-fix-delete-update-article-162888665`
- Install the required dependencies:
     `npm start`
should be able to:
- update an article
- be represented with a dialog box on delete
- not see a delete/update article button if you aren't the article author
 
#### What are the relevant pivotal tracker stories?
[#162888665](https://www.pivotaltracker.com/story/show/162888665)

![image](https://user-images.githubusercontent.com/30722290/50566056-654e1a80-0d46-11e9-9996-c12a65b2d386.png)
![image](https://user-images.githubusercontent.com/30722290/50566059-667f4780-0d46-11e9-92b5-3774c556b629.png)
![image](https://user-images.githubusercontent.com/30722290/50566061-6aab6500-0d46-11e9-952e-4e6cfac371f8.png)
![image](https://user-images.githubusercontent.com/30722290/50566062-6da65580-0d46-11e9-9aa9-dcfe4f9dc6d5.png)
![image](https://user-images.githubusercontent.com/30722290/50566069-7008af80-0d46-11e9-925f-55e5e90b3f29.png)

#### Checklist:
